test/typedoc_coverage_test.ts

```diff
@@ -54,10 +57,14 @@ function getReflectionData() {
 
 function traverse(data: ReflectionData): ReflectionData[] {
   const entitiesWithMissingTypedoc: ReflectionData[] = [];
-  if (!hasComment(data) && data.kindString !== 'Project') {
+  // TODO: Add comments for all types in development.ts and remove this condition.
```

Comment:
Seems best to make sure there's a task filed for this TODO so it doesn't get forgotten or lost

Reply:
Create o/pak/839.